src/reactViews/iq/pages/chatscreen/IqChat.tsx

```diff
@@ -0,0 +1,602 @@
+      value: {
+        enabled: true,
+      },
+    });
+
+    tsvscode.postMessage({
+      command: "vscode-couchbase.iq.showNewChatButton",
+      value: {
+        enabled: true,
+      },
+    });
+  }, []);
+
+  const handleMessageLike = (index: number, qaId: string) => {
+    const originalReply = messages.userChats[index].message;
+    const originalQuestion =
+      index - 1 > 0 ? messages.userChats[index - 1].message : "";
+    const newMessage: userMessage = {
+      message:
+        "Glad you liked the result. Would you like to give more feedback",
+      sender: "feedback",
+      msgDate: (Date.now() / 1000).toFixed(0),
+      qaId: qaId,
+      feedbackSent: false,
+    };
+
+    const messagesCopy = [...messages.userChats];
+    messagesCopy[index].feedbackSent = true;
+
+    const updatedMessages = [...messagesCopy, newMessage];
+    setMessages({
+      chatId: messages.chatId,
+      userChats: updatedMessages,
+    });
+
+    setActions([
+      ChatAction(
+        "Send Feedback",
+        setShowFeedbackModal,
+        setFeedbackModalData,
+        index,
+        qaId
+      ),
+    ]);
+    // send info to lambda
+    tsvscode.postMessage({
+      command: "vscode-couchbase.iq.sendFeedbackPerMessageEmote",
+      value: {
+        type: "like",
+        question: originalQuestion,
+        reply: originalReply,
+        msgDate: newMessage.msgDate,
+        additionalFeedback: "",
+        qaId: qaId,
+        chatId: messages.chatId,
+        orgId: org.data.id,
+        userChats: updatedMessages,
+      },
+    });
+  };
+
+  const handleMessageDislike = (index: number, qaId: string) => {
+    const originalReply = messages.userChats[index].message;
+    const originalQuestion =
+      index - 1 > 0 ? messages.userChats[index - 1].message : "";
+    const newMessage: userMessage = {
+      message:
+        "Oh! We are very sorry. Can you please give us additional info via feedback",
+      sender: "feedback",
+      msgDate: (Date.now() / 1000).toFixed(0),
+      qaId: qaId,
+      feedbackSent: false,
+    };
+
+    const messagesCopy = [...messages.userChats];
+    messagesCopy[index].feedbackSent = true;
+
+    const updatedMessages = [...messagesCopy, newMessage];
+    setMessages({
+      chatId: messages.chatId,
+      userChats: updatedMessages,
+    });
+
+    // set actions to feedback
+    setActions([
+      ChatAction(
+        "Send Feedback",
+        setShowFeedbackModal,
+        setFeedbackModalData,
+        index,
+        qaId
+      ),
+    ]);
+
+    tsvscode.postMessage({
+      command: "vscode-couchbase.iq.sendFeedbackPerMessageEmote",
+      value: {
+        type: "dislike",
+        question: originalQuestion,
+        reply: originalReply,
+        msgDate: newMessage.msgDate,
+        additionalFeedback: "",
+        qaId: qaId,
+        chatId: messages.chatId,
+        userChats: updatedMessages,
+      },
+    });
+    // send info to lambda
```

Comment:
```suggestion
    // send info to lambda
```
Put it at correct place

Reply:
done